scalafix/rules/src/main/scala/fix/Scalacollectioncompat_newcollections.scala

```diff
@@ -66,6 +66,11 @@ case class Scalacollectioncompat_newcollections(index: SemanticdbIndex)
       Symbol("_root_.scala.collection.mutable.SetLike.retain.")
     )
 
+  val mapZip = 
+    SymbolMatcher.exact(
+      Symbol("_root_.scala.collection.IterableLike#zip(Lscala/collection/GenIterable;Lscala/collection/generic/CanBuildFrom;)Ljava/lang/Object;.")
```

Comment:
It looks like it’s going to match *any* call to `zip`, even if the receiver is not of type `Map[_, _]`.

Reply:
then we are blocked by https://github.com/scalameta/scalameta/issues/1212